src/components/RangeInput.vue

```diff
@@ -1,15 +1,20 @@
 <template>
 <div v-if="state" :class="suit()">
-  <form :class="suit('form')" @submit.prevent="refine()">
+  <slot
+    :currentRefinements="this.state.start"
+    :refine="this.refine"
+  >
+  <form :class="suit('form')" @submit.prevent="refine(minInput, maxInput)">
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'min')]" type="number" ref="minInput" :min="this.min" :placeholder="this.min"/>
+      <input :class="[suit('input'), suit('input', 'min')]" type="number" :min="this.min"  :placeholder="this.min" :value="this.state.start && this.state.start[0]" @change="minInput = $event.currentTarget.value"/>
```

Comment:
The behaviour is different from the React & IS version, the values should be clamped to the range. It's not implemented in the connector for backward compatibility issue. But the behaviour exists in the [widget](https://github.com/algolia/instantsearch.js/blob/cd8f17ecf5b6e7aef14a54a9bec51fa8cd167f69/src/widgets/range-input/range-input.js#L42-L45).